.github/workflows/release.yml

```diff
@@ -15,7 +15,36 @@ jobs:
   release:
     name: Run "Release Please"
     runs-on: ubuntu-22.04
+    outputs:
+      config_package_published:
+        ${{ steps.release.outputs.config-inject--release_created }}
     steps:
       - uses: google-github-actions/release-please-action@v3.7.8
+        id: release
         with:
           command: manifest
+  publish:
+    name: Publish to GPR
+    runs-on: ubuntu-22.04
+    needs: release
+    if: ${{ needs.release.outputs.config_package_published }}
+    defaults:
+      run:
+        working-directory: config-inject
+    steps:
+      - uses: actions/checkout@v3.5.1
+      - uses: pnpm/action-setup@v2.2.4
+        with:
+          version: 8
+      - uses: actions/setup-node@v3.6.0
+        with:
+          node-version: "16.x"
+          scope: "@pleo-io"
+          cache: "pnpm"
+          cache-dependency-path: config-inject/pnpm-lock.yaml
+      - run: pnpm install --frozen-lockfile --ignore-scripts
+      - run: pnpm run build
+      - run: npm publish
```

Comment:
This will probably not work.

As least I fought this a lot and ended up with this: https://github.com/pleo-io/lambda-bff-setup/blob/main/.github/workflows/release.yml#L45

Reply:
But that's exactly what `NODE_AUTH_TOKEN` does 🤔 https://github.com/actions/setup-node/blob/d98fa1113850e562f83c7fc3a89c0ecd7a87fbed/src/authutil.ts#L47-L58

Reply:
Agree. I gave up finding out why it didn't work – tried a lot of different things... They [failed like this](https://github.com/pleo-io/lambda-bff-setup/actions/runs/4536322353/jobs/7992870052. But if we have multiple npm packages here we need the publish script. 

Context:
- failed attempt 1 https://github.com/pleo-io/lambda-bff-setup/commit/9eea686e5d246a50ee35dff87611a895a5e13583
- failed attempt 2 https://github.com/pleo-io/lambda-bff-setup/commit/6fff79bd1cee7c2f7ae731d675d92130bb60ae85
- what fixed it https://github.com/pleo-io/lambda-bff-setup/commit/683153dd6b7dae39ab9b8a84ad9b03442a009256

Reply:
If NODE_AUTH_TOKEN works here then all good.

Reply:
I'll play around to see if it's needed